recipes-core/acrn/acrn-hypervisor.bb

```diff
@@ -4,11 +4,11 @@ ACRN_BOARD ?= "nuc7i7dnb"
 ACRN_FIRMWARE ?= "uefi"
 ACRN_SCENARIO  ?= "sdc"
 
-EXTRA_OEMAKE += "HV_OBJDIR=${B}/hypervisor EFI_OBJDIR=${B}/efi-stub"
+EXTRA_OEMAKE += "T=${S} HV_OBJDIR=${B}/hypervisor EFI_OBJDIR=${B}/efi-stub"
 EXTRA_OEMAKE += "BOARD=${ACRN_BOARD} FIRMWARE=${ACRN_FIRMWARE} SCENARIO=${ACRN_SCENARIO}"
 EXTRA_OEMAKE += "BOARD_FILE=${S}/misc/acrn-config/xmls/board-xmls/${ACRN_BOARD}.xml SCENARIO_FILE=${S}/misc/acrn-config/xmls/config-xmls/${ACRN_BOARD}/${ACRN_SCENARIO}.xml"
 
-inherit python3native deploy
+inherit python3native deploy cml1
```

Comment:
Nice touch to inherit cml1. I had previously just copied and pasted it into the recipe :)